### PR TITLE
feat(backend): design html emails and adjust texts

### LINKED ISSUE #2466: 🚀 [Feature] Design HTML E-Mails And Adjust Texts
## 🚀 Feature

Design HTML e-mails and adjust texts with @BerndHueckstaedt and @Hardy-Hart-am-Start .

### Todo

- [ ] have the Gradido logo?
- [ ] add imprint

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Design HTML E-Mails And Adjust Texts.

### Issues

- fixes #2466

### Todo

- [x] `CONFIG`:
  - [x] shall `SUPPORT_MAIL` be included and named `COMMUNITY_SUPPORT_MAIL`?
    - [x] backend
    - [x] frontend
    - [x] bare_metal
- [x] design
  - [x] create new issue for further design: #2473
  - [x] imprint
    - [x] with logo? (`frontend/img/brand/green.png`)
- [x] greeting formula:
  - [x] indivdual for each community:
    - [x] shall a new community change the locales?
- [x] add imprint:
  - [x] every community has its own imprint says @BerndHueckstaedt . so we make:
    - [x] explicit imprint in the locales?
    - [ ] link to the imprint?
      - if so do we put the URL into the
        - [ ] emails PUG file to be able to change the language part?
        - [ ] backend locales?
        - [ ] config?
- [x] (no) move `https://gradido.net/de/contact/` to locales?
  - [x] (yes) or shall it be replaced by the support email?
- [x] adjust tests

### CODE REVIEW

**Comment by ulfgebhardt** [comment]
@Tirokk Please consider to wait for the fake email server #2441 to be able to generate the emails more easily. Ask @mahula for details and merge shedule

**Comment by Tirokk** [comment]
> @Tirokk Please consider to wait for the fake email server #2441 to be able to generate the emails more easily. Ask @mahula for details and merge shedule

I do not understand. Why? @ulfgebhardt 
There are no problems to generate the emails except to spend some time to go through all functions which are sending emails.

I have already done this and have the go for all emails from @BerndHueckstaedt and we can merge this after approve.

File: backend/src/locales/de.json
Comment (ulfgebhardt) [inline backend/src/locales/de.json:17]:
```suggestion
      "ifYouAreNotTheOne": "Wenn du nicht derjenige bist, der versucht hat sich erneut zu registrieren, wende dich bitte an unseren Support:",
```

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**
